support-frontend/assets/components/subscriptionCheckouts/subscriptionsSurvey/SubscriptionsSurvey.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import type { SubscriptionProduct } from 'helpers/subscriptions';
+
+import AnchorButton from 'components/button/anchorButton';
+import Text from 'components/text/text';
+import Content from 'components/content/content';
+import './subscriptionsSurvey.scss';
+
+type PropTypes = {|
+  product: SubscriptionProduct,
+|};
+
+const surveyLinks: {[key: SubscriptionProduct]: string} = {
+  DigitalPack: 'https://www.surveymonkey.co.uk/r/QF9ZGQR',
+  GuardianWeekly: 'https://www.surveymonkey.co.uk/r/QFNYV5G',
+  Paper: 'https://www.surveymonkey.co.uk/r/Q37XNTV',
+};
+
+export const SubscriptionsSurvey = ({ product }: PropTypes) => {
+  const surveyLink = surveyLinks[product];
+  const title = 'Tell us about your subscription';
+  const message = 'Please fill out this short form to help us learn a little more about why you purchased a subscription';
```

Comment:
this does seema bit verbose, I'm sure some of the words can be deleted without changing the meaning (although maybe it came from someone who knows more than me!)

Reply:
also short form is not clear, perhaps you could say something quantified eg "2 minute" or "4 question"